app/controllers/deploys_controller.rb

```diff
@@ -99,6 +99,7 @@ def new
   def create
     deploy_service = DeployService.new(current_user)
     @deploy = deploy_service.deploy!(stage, deploy_params)
+    Samson::Hooks.fire(:post_ledger_event, @deploy) if ENV["RAILS_ENV"] != 'test'
```

Comment:
hooks should be universal ... reuse before_deploy or after_deploy ?
... and always fire them ...